doc/design/garbage-collection-assets.md

```diff
@@ -0,0 +1,40 @@
+# Asset Garbage Collection
+
+## Background
+
+Now that the [design for asset blob and upload garbage collection](./garbage-collection-uploads-asset-blobs-2.md) is deployed to staging, we are ready to implement Asset garbage collection. The garbage collection process in this design document only applies to “regular” assets; garbage collection of zarrs is not covered and is out of scope for this design doc.
+
+## What is an “Orphaned Asset”
+
+It’s important to precisely define what it means for an asset to be eligible for garbage collection (an “orphaned asset”).
+An orphaned asset has the following properties:
+- The asset is not currently associated with any Versions
+- The asset is older than 30 days
+
+## Integration with existing garbage collection routines
+
+Garbage collection routines for Asset Blobs and Uploads are already implemented and deployed (though not run yet). Currently, the number of Asset Blobs and Uploads that need to be garbage collected is not substantial. However, the act of running the Asset garbage collection routine will result in a significant amount of Asset Blobs being orphaned, which can then in turn be collected by the Asset Blob garbage collection routine.
+
+## Implementation notes
+
+A new module, `asset`, will be added to the `garbage_collection` service (see `../dandiapi/api/services/garbage_collection/`). This module will mirror the structure of the sibling `asset_blob` and `upload` modules and will be responsible for deleting orphaned assets and handling associated recordkeeping (i.e. adding a record to the `GarbageCollectionEvent` table for every delete event).
```

Comment:
just to make sure -- we make conscious decision to **not** add any record to audit since it is not user/API invoked action, correct?